src/components/DetailPage/DetailPage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import MeredithGrey from '../../assets/doctors/MeredithGrey.jpeg';
+// import PropTypes from 'prop-types';
```

Comment:
- [ ] Let's make sure we remove unused code

Reply:
Sure, I'm going to.